components/ui/TypedWord.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="typed-word__container" :class="{ 'typed-word__cursor': showCursor }">
```

Comment:
The top-level element should include a BEM block class named after the component so this element should include the class `typed-word`. I argue that there is no need for adding the container too although you can try to convince me. ;)

Reply:
Correct me if I'm wrong, please: the class `typed-word__cursor` is there for signalling the cursor is visible, is it right? If that's the case, consider converting it into a [boolean BEM block modifier](https://en.bem.info/methodology/quick-start/#boolean) like `typed-word_cursor-visible`.